docs/source/getting-started.rst

```diff
@@ -388,12 +388,11 @@ Set the above settings in your ``jupyterhub_config``:
 .. code:: python
 
    # Tell JupyterHub to create system accounts
-   from oauthenticator.globus import LocalGlobusOAuthenticator
-   c.JupyterHub.authenticator_class = LocalGlobusOAuthenticator
-   c.LocalGlobusOAuthenticator.enable_auth_state = True
-   c.LocalGlobusOAuthenticator.oauth_callback_url = 'https://[your-host]/hub/oauth_callback'
-   c.LocalGlobusOAuthenticator.client_id = '[your app client id]'
-   c.LocalGlobusOAuthenticator.client_secret = '[your app client secret]'
+   from oauthenticator.globus import GlobusOAuthenticator
+   c.JupyterHub.authenticator_class = GlobusOAuthenticator
+   c.GlobusOAuthenticator.oauth_callback_url = 'https://[your-host]/hub/oauth_callback'
+   c.GlobusOAuthenticator.client_id = '[your app client id]'
+   c.GlobusOAuthenticator.client_secret = '[your app client secret]'
```

Comment:
The docs here tweak a couple things. `GlobusOAuthenticator` is probably better since it assumes less about the context the admin is using the Authenticator. In Zero to Jupyterhub, for example, I don't think LocalGlobusOAuthenticator is useful. This makes the config a little more copy-pastable.

I also removed the `c.LocalGlobusOAuthenticator.enable_auth_state = True`, since it isn't needed in the basic configuration, only when injecting tokens into user notebooks.